src/test/moves/disable.test.ts

```diff
@@ -0,0 +1,109 @@
+import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
+import GameManager from "../utils/gameManager";
+import { Abilities } from "#enums/abilities";
+import { Moves } from "#enums/moves";
+import { Species } from "#enums/species";
+import { CommandPhase, EnemyCommandPhase, TurnInitPhase } from "#app/phases.js";
+import { Mode } from "#app/ui/ui.js";
+import { getMovePosition } from "../utils/gameManagerUtils";
+import {Command} from "#app/ui/command-ui-handler";
+import { MoveResult } from "#app/field/pokemon.js";
+import { Stat } from "#app/data/pokemon-stat.js";
+import { DisabledTag } from "#app/data/battler-tags.js";
+import { BattlerTagType } from "#enums/battler-tag-type";
+
+describe("Moves - Disable", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  /** If in CommandPhase, input a move */
+  const _useMove = (move?: Moves) => {
+    move ??= Moves.DISABLE;
+
+    game.onNextPrompt("CommandPhase", Mode.COMMAND, () => {
+      game.scene.ui.setMode(Mode.FIGHT, (game.scene.getCurrentPhase() as CommandPhase).getFieldIndex());
+    });
+    game.onNextPrompt("CommandPhase", Mode.FIGHT, () => {
+      const movePosition = getMovePosition(game.scene, 0, move);
+      (game.scene.getCurrentPhase() as CommandPhase).handleCommand(Command.FIGHT, movePosition, false);
+    });
+  };
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+    game.override.battleType("single");
+    game.override.ability(Abilities.NONE);
+    game.override.enemyAbility(Abilities.NONE);
+    game.override.moveset([Moves.DISABLE, Moves.SPLASH, Moves.SPLASH, Moves.SPLASH]);
+    game.override.enemyMoveset([Moves.SPLASH,Moves.SPLASH,Moves.SPLASH,Moves.SPLASH]);
```

Comment:
Use one of the test utils instead:
```suggestion
    game.override.enemyMoveset(ALL_SPLASH);
```

Reply:
It's `SPLASH_ONLY` btw.